content/blog/grpc-web-interceptor.md

```diff
@@ -209,6 +163,31 @@ TestStreamInterceptor.prototype.intercept = function(request, invoker) {
 };
 ```
 
+## Binding interceptors
+
+By passing an array of inteceptor instances using an appropriate option key,
+you can bind interceptors to a client when the client is instantiated:
+
+```js
+const promiseClient = new FooServicePromiseClient(
+    host, creds, {'unaryInterceptors': [interceptor1, interceptor2, interceptor3]});
+
+const client = new FooServiceClient(
+    host, creds, {'streamInterceptors': [interceptor1, interceptor2, interceptor3]});
+```
+
+{{< note >}}
+  Interceptors are executed in reverse order for request processing, and in order for response processing:
+
+  ```nocode
+  Request  --> interceptor3  --> interceptor2  --> interceptor1 --> Intercepted --> Server
+                                                                    request           |
+                                                                                      |
+  Intercepted <-- interceptor3 <-- interceptor2 <-- interceptor1 <--   Response  <-- Server
+  response
+  ```
```

Comment:
This ASCII art seems clunky, but I wasn't sure what to replace it buy, so I've left it in for now.

Might you be open to a more textual description that explains the order processing using a functional notation? Possibly something like this:

> To illustrate this, assume that `i1()`, `i2()` and `i3()` functions representing the effects of the interceptors bound to the client.
> - If `req` is the original client request, then the server will receive the request created by the following interceptor transformations: `i1(i2(i3(req)`.
> - Similarly, if `resp` is a server response, then the client will receive the response that results from the following order of interceptor processing: `i3(i2(i1(resp)`.

Reply:
Sure. Or can we add an image to the blog?

Reply:
Sure an image would be ok. Maybe something like this quick sketch (with all the labels added)?

![image](https://user-images.githubusercontent.com/4140793/84531235-47624480-acb2-11ea-8a96-230c5eba15e3.png)

If you prefer an image, I'd let you create it. If you're ok with the text suggested in the previous comment in this thread, let me know.

Reply:
Yeah I'll create an image. Where should I submit it to?

Reply:
You can submit a PR over the `blog-grpc-web` branch. Place the image under `static/img`. Thanks!

Reply:
Thanks for the image.